mensa/settings.py

```diff
@@ -27,7 +27,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ["*"]
```

Comment:
```suggestion
ALLOWED_HOSTS = []
```

Is this change necessary? `ALLOWED_HOSTS` shouldn't matter in debug mode.

Reply:
No this is definitely not necessary, I didn't mean to commit/push this change... Therefore I reverted this change immediately. I wanted to access the website from my phone. Since in debug the server rejected the phone's (non-`localhost`) request I changed this line to make it work.